packages/salesforce-adapter/src/adapter_creator.ts

```diff
@@ -249,14 +250,24 @@ export const createDeployProgressReporter = async (
     return ` View ${deploymentOrValidation} status [in Salesforce](${deploymentUrl})`
   }
 
+  const linkToSalesforceDeploymentsPage = (): string => {
+    if (!baseUrl) {
+      return ''
+    }
+    return ` View deployments [in Salesforce](${baseUrl}lightning/setup/DeployStatus/home)`
+  }
```

Comment:
Nit: why is this a function?
```suggestion
  const salesforceDeploymentsPageLinkText = baseUrl ? ` View deployments [in Salesforce](${baseUrl}lightning/setup/DeployStatus/home)` : ''
```